src/components/ProductCard/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { StarRate, Wishlist, Basket } from "@shared";
+import { Link } from "react-router-dom";
+
+interface ProductProp {
+  img: string;
+  name: string;
+  type: string;
+  cost: number;
+  rate: number;
+}
+
+const ProductCardExample: ProductProp = {
+  img: "https://www.collinsdictionary.com/images/full/paper_111691001.jpg",
+  name: "Paper name",
+  type: "Paper type",
+  cost: 1500,
+  rate: 1,
+};
+
+export default function ProductCard ({ img, name, type, cost, rate }: ProductProp & typeof ProductCardExample): JSX.Element {
```

Comment:
Use only interface. For ProductCard we do not need to set default values.
Instead - create mock of cards in(out)side Main component and .map() them.